src/features/events/store.ts

```diff
@@ -93,6 +128,68 @@ const eventsSlice = createSlice({
       state.eventList = remoteList(action.payload);
       state.eventList.loaded = new Date().toISOString();
     },
+    filterTextAdded: (
+      state,
+      action: PayloadAction<{
+        filterText: string;
+      }>
+    ) => {
+      const { filterText } = action.payload;
+      state.filters.text = filterText;
+    },
+    filterUpdated: (
```

Comment:
I'm sure this can be refactored to be simpler, but this is working fine so lets not spend time on that now.

Reply:
Actually, lets keep working on this because I think it's a good learning experience.